content/raw/chat/online_users.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Online Users spec"
+version: "1.0.0"
+description: ""
+date: 2020-10-06T08:48:57+00:00
+lastmod: 2020-10-06T08:48:57+00:00
+draft: false
+images: []
+menu:
+  raw:
+    parent: "chat"
+weight: 100
+toc: true
+---
+
+# Online Users
+
+## Motivation
+
+It's important for end users to know which community members are online or offline, they can use this to judge how active a community is, if a member is likely following a discussion and whenever to message that member or not. At the same time introducing such a feature might have privacy concerns which we need to be aware of, so this feature is implemented as a opt-in functionality (see notes).
+
+## Designs
+
+* [member section](https://www.figma.com/file/Mr3rqxxgKJ2zMQ06UAKiWL/%F0%9F%92%AC-Chat%E2%8E%9CDesktop?node-id=6644%3A46846)
+* [status selector](https://www.figma.com/file/IPpvkpDWabBKJTeo6bFop0/Kuba%E2%8E%9CDesktop?node-id=106%3A20383)
+
+## Use Cases
+
+### Setting state as online
```

Comment:
Shouldn't these two sections ("Setting state as online" and "Setting state as offline") be removed from this spec as they are dealt with more comprehensively in the "feat: online users spec with status set automatically" spec?  Then this spec could focus only on the right hand user bar, and could ignore the mechanics of how a user is determined to be either Online or Inactive (as this is covered in that other spec)?